queries.sql

```diff
@@ -23,3 +23,32 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'Digmon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
```

Comment:
After deleting. `COUNT(*)` them from animals then you can rollback again.

Reply:
![Screen Shot 2022-10-19 at 17 01 15](https://user-images.githubusercontent.com/69011963/196685138-a1e8ef9a-05ad-48f7-96c3-ed3ca78fd937.png)

Reply:
Yes Exactly, you did a great job. However, these are the requirements that shown to us.